test/test_helper.rb

```diff
@@ -3,6 +3,7 @@
 require 'simplecov'
 require 'simplecov-lcov'
 SimpleCov::Formatter::LcovFormatter.config.report_with_single_file = true
+SimpleCov::Formatter::LcovFormatter.config.lcov_file_name = 'coverage.lcov'
```

Comment:
My assumption is that this filename matches the one being defined in the shared workflow?